### PR TITLE
feat: mobile collection view

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11627](https://app.clickup.com/t/2645698/DEV-11627)

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11627 Adjust filters to mobile on collection view](https://app.clickup.com/t/2645698/DEV-11627)

File: components/ModalTemplateMobileCollectionFilters.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<script setup lang="ts">
+import type { FiltersAttribute } from '@/types/filters'
+import type { SelectStringOption } from '@lukso/web-components'
+
+const { formatMessage } = useIntl()
+const {
+  filters,
+  setFilters,
+  attributeFilterOptions,
+  //   // attributeFilterValues,
```
Comment (doubleppereira) [inline components/ModalTemplateMobileCollectionFilters.vue:10]: Should we remove this?

Comment (dzbo) [reply components/ModalTemplateMobileCollectionFilters.vue:10]: yep

**Review by Prozqa — APPROVED** [review]
image
image
image
Looks good!

**Pull request merged**